Product/Production/Common/Properties/src/main/resources/exchangeInfo.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<exchangeInfo xmlns="urn:gov:hhs:fha:nhinc:exchange" xmlns:ns2="urn:gov:hhs:fha:nhinc:exchange:directory">
+    <lastUpdated></lastUpdated>
```

Comment:
Optional: We need to migrate all constant variable in gateway.properties that relate to Uddi and mapping into this xml